instance-flavor-check

```diff
@@ -17,54 +17,6 @@
 # You should have received a copy of the GNU General Public License along with
 # instance-billing-flavor-check. If not, see <http://www.gnu.org/licenses/>.
 
-import argparse
-import logging
-import sys
+from instance_billing_flavor_check.utils import check_payg_byos
```

Comment:
This file should have `#!/usr/bin/python3`

Reply:
OK, it has it already